service/src/main/resources/db/changesets/20230427.yaml

```diff
@@ -0,0 +1,32 @@
+# Add jobs table
```

Comment:
super minor, but could you update this comment or remove it (since there's the changeSet.id field already with a description)?

Reply:
oops!  I'll get rid of that comment